docs/_articles/en/soql/soql-tasks.md

```diff
@@ -0,0 +1,31 @@
+---
+title: SOQL Tasks
+lang: en
+---
+
+This is a repository of sample tasks using SOQL that you can complete in Code Builder or VS Code Extensions for desktop. We’ll keep adding to this list, so make sure you visit this topic often. 
+
+
+## Use the SOQL Query Editor Create a Query To List Accounts with a New York State Address
+
+1. From the command palette, run **SFDX: Create Query in SOQL Builder** to open SOQL Query Builder.
+2. Click **File > Save** to save the query. Make sure to retain the .soql file extension.
+3. Click the Switch Between SOQL Builder and Text Editors icon (<img src="../../../images/go-to-file.svg">) to reopen the SOQL Query Builder.
+4. In the `From` field, search object, and select the `Account` object.
+5. In `Fields`, select `Name`, `BillingState`, and `BillingCountry`.
+6. In the `Filter` field, select `AND`, and set:
+    `BillingCountry = USA`
+    `AND`
+    `BillingState = NY`
+7.   Click **Run Query**.
+
+A Query Result tab pops up. You can then save the result in `csv` or `json` formats.
+
+
+## Resources
+
+- [Example SELECT Clauses](https://developer.salesforce.com/docs/atlas.en-us.236.0.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_select_examples.htm)
+- [Salesforce Object Query Language (SOQL)](https://developer.salesforce.com/docs/atlas.en-us.236.0.soql_sosl.meta/soql_sosl/sforce_api_calls_soql.htm)
+
+Trailhead
```

Comment:
Can you rearrange these resources to be just 3 bullets, with format similar to how we do it in core RNs?  i.e. 

* Guide: Topic 
* Trailhead: Topic